### ISSUE TITLE
Error during start of VTMgo add-on

### ISSUE THREAD

**Issue by benvanloock** [add-ons/plugin.video.vtm.go#208]
**Describe the bug**
When (re-)entering the add-on, Kodi shows there was an error, with more information to be found in the log file.
The kodi.log file shows 'InvalidLoginException'  (see below). The used login should be ok. I verified with the account with my phone and on my desktop.

**To Reproduce**
After installing the VTMgo add-on, configuring a user, the add-on seems to work fine. I can select and see what I want.
However, after leaving the add-on and re-entering it, I always got the message there was 'an error' and can not enter the add-on. 

**System**
 - Addon version: 1.1.2
 - Kodi version:  18.7
 - Inputstream adaptive version: .2.4.5
 - Operating System: Open Source Media Center 2020.06-1 kernel: Linux ARM 32-bit version 4.19.122-1-osmc
 - Special Hardware: RPI

**Additional context**

the kodi.log :
```
2020-09-27 18:17:51.568 T:1637835488  NOTICE: UDP: Listening on port 9777 (ipv6 : true)
2020-09-27 18:17:51.889 T:1732240096  NOTICE: Register - new cec device registered on cec->RPI: CEC Adapter (2708:1001)
2020-09-27 18:18:09.920 T:1679799008   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <class 'resources.lib.vtmgo.vtmgoauth.InvalidLoginException'>
                                            Traceback (most recent call last):
                                              File "/home/osmc/.kodi/addons/plugin.video.vtm.go/service.py", line 8, in <module>
                                                service.run()
                                              File "/home/osmc/.kodi/addons/plugin.video.vtm.go/resources/lib/service.py", line 193, in run
                                                BackgroundService().run()
                                              File "/home/osmc/.kodi/addons/plugin.video.vtm.go/resources/lib/service.py", line 27, in __init__
                                                self.vtm_go = VtmGo(self._kodi)
                                              File "/home/osmc/.kodi/addons/plugin.video.vtm.go/resources/lib/vtmgo/vtmgo.py", line 278, in __init__
                                                self._authenticate()
                                              File "/home/osmc/.kodi/addons/plugin.video.vtm.go/resources/lib/vtmgo/vtmgo.py", line 288, in _authenticate
                                                token = self._auth.get_token()
                                              File "/home/osmc/.kodi/addons/plugin.video.vtm.go/resources/lib/vtmgo/vtmgoauth.py", line 89, in get_token
                                                self._token = self._login()
                                              File "/home/osmc/.kodi/addons/plugin.video.vtm.go/resources/lib/vtmgo/vtmgoauth.py", line 136, in _login
                                                raise InvalidLoginException()
                                            InvalidLoginException
                                            -->End of Python script error report<--
2020-09-27 18:18:10.164 T:1520583392   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <class 'resources.lib.vtmgo.vtmgoauth.InvalidLoginException'>
                                            Traceback (most recent call last):
                                              File "/home/osmc/.kodi/addons/plugin.video.vtm.go/plugin.py", line 18, in <module>
                                                plugin.run(sys.argv)
                                              File "/home/osmc/.kodi/addons/plugin.video.vtm.go/resources/lib/plugin.py", line 218, in run
                                                routing.run(params)
                                              File "/home/osmc/.kodi/addons/script.module.routing/lib/routing.py", line 130, in run
                                                self._dispatch(self.path)
                                              File "/home/osmc/.kodi/addons/script.module.routing/lib/routing.py", line 141, in _dispatch
                                              File "/home/osmc/.kodi/addons/plugin.video.vtm.go/resources/lib/plugin.py", line 29, in index
                                                select_profile()
                                              File "/home/osmc/.kodi/addons/plugin.video.vtm.go/resources/lib/plugin.py", line 44, in select_profile
                                                Authentication(kodi).select_profile(key)
                                              File "/home/osmc/.kodi/addons/plugin.video.vtm.go/resources/lib/modules/authentication.py", line 23, in __init__
                                                self._vtm_go = VtmGo(self._kodi)
                                              File "/home/osmc/.kodi/addons/plugin.video.vtm.go/resources/lib/vtmgo/vtmgo.py", line 278, in __init__
                                                self._authenticate()
                                              File "/home/osmc/.kodi/addons/plugin.video.vtm.go/resources/lib/vtmgo/vtmgo.py", line 288, in _authenticate
                                                token = self._auth.get_token()
                                              File "/home/osmc/.kodi/addons/plugin.video.vtm.go/resources/lib/vtmgo/vtmgoauth.py", line 89, in get_token
                                                self._token = self._login()
                                              File "/home/osmc/.kodi/addons/plugin.video.vtm.go/resources/lib/vtmgo/vtmgoauth.py", line 136, in _login
                                                raise InvalidLoginException()
                                            InvalidLoginException
                                            -->End of Python script error report<--
2020-09-27 18:18:10.370 T:1528976096   ERROR: GetDirectory - Error getting plugin://plugin.video.vtm.go/
2020-09-27 18:18:10.374 T:1915483696   ERROR: CGUIMediaWindow::GetDirectory(plugin://plugin.video.vtm.go/) failed
2020-09-27 18:18:41.535 T:1915483696   ERROR: Control 50 in window 10025 has been asked to focus, but it can't
2020-09-27 18:18:45.552 T:1790329568   ERROR: EXCEPTION Thrown (PythonToCppException) : -->Python callback/script returned the following error<--
                                             - NOTE: IGNORING THIS CAN LEAD TO MEMORY LEAKS!
                                            Error Type: <class 'resources.lib.vtmgo.vtmgoauth.InvalidLoginException'>
                                            Traceback (most recent call last):
                                              File "/home/osmc/.kodi/addons/plugin.video.vtm.go/plugin.py", line 18, in <module>
                                                plugin.run(sys.argv)
                                              File "/home/osmc/.kodi/addons/plugin.video.vtm.go/resources/lib/plugin.py", line 218, in run
                                                routing.run(params)
                                              File "/home/osmc/.kodi/addons/script.module.routing/lib/routing.py", line 130, in run
                                                self._dispatch(self.path)
                                              File "/home/osmc/.kodi/addons/script.module.routing/lib/routing.py", line 141, in _dispatch
                                                view_func(**kwargs)
                                              File "/home/osmc/.kodi/addons/plugin.video.vtm.go/resources/lib/plugin.py", line 29, in index
                                                select_profile()
                                              File "/home/osmc/.kodi/addons/plugin.video.vtm.go/resources/lib/plugin.py", line 44, in select_profile
                                                Authentication(kodi).select_profile(key)
                                              File "/home/osmc/.kodi/addons/plugin.video.vtm.go/resources/lib/modules/authentication.py", line 23, in __init__
                                                self._vtm_go = VtmGo(self._kodi)
                                              File "/home/osmc/.kodi/addons/plugin.video.vtm.go/resources/lib/vtmgo/vtmgo.py", line 278, in __init__
                                                self._authenticate()
                                              File "/home/osmc/.kodi/addons/plugin.video.vtm.go/resources/lib/vtmgo/vtmgo.py", line 288, in _authenticate
                                                token = self._auth.get_token()
                                              File "/home/osmc/.kodi/addons/plugin.video.vtm.go/resources/lib/vtmgo/vtmgoauth.py", line 89, in get_token
                                                self._token = self._login()
                                              File "/home/osmc/.kodi/addons/plugin.video.vtm.go/resources/lib/vtmgo/vtmgoauth.py", line 136, in _login
                                                raise InvalidLoginException()
                                            InvalidLoginException
                                            -->End of Python script error report<--
2020-09-27 18:18:45.742 T:1610609376   ERROR: GetDirectory - Error getting plugin://plugin.video.vtm.go/
2020-09-27 18:18:45.744 T:1915483696   ERROR: CGUIMediaWindow::GetDirectory(plugin://plugin.video.vtm.go/) failed
2020-09-28 05:56:12.146 T:1761489632  NOTICE: VideoInfoScanner: Starting scan ..
```

**Comment by dagwieers** [comment]
I think this is a duplicate of #195 and may be fixed in PR #207

**Comment by benvanloock** [comment]
Tx for the feedback, Dag !

As mentioned in issue #195 I installed the zip https://github.com/add-ons/plugin.video.vtm.go/archive/profile-fix.zip .
After explicitly setting again my password and then starting the add-on, it started to work. I left the add-on, started another one and came back to VTMgo. And it still worked. I'll test again later today.

**Comment by michaelarnauts** [comment]
Fixed in #207
